widget/table.go

```diff
@@ -226,6 +226,14 @@ func (c *tableCells) CreateRenderer() fyne.WidgetRenderer {
 	return &tableCellsRenderer{cells: c, pool: &syncPool{}, visible: make(map[cellID]fyne.CanvasObject)}
 }
 
+func (c *tableCells) Resize(s fyne.Size) {
+	if s == c.size {
+		return
+	}
+	c.BaseWidget.Resize(s)
```

Comment:
Not sure if this is needed - it checks the size is different (which is done above on line 230), updates the size, and calls Layout (which does nothing in tableCells). Besides locking, I don't see why this call is needed, am I missing something?

Reply:
It was just a best practice I thought - we have overridden BaseWidget so we propagate.
That function could do other things in the future.